src/screens/DetailGroupImage/index.js

```diff
@@ -86,32 +80,32 @@ const styles = StyleSheet.create({
     justifyContent: 'flex-end',
     alignItems: 'flex-end',
     paddingBottom: 10,
-    paddingRight: 10,
+    paddingRight: 10
   },
   container: {
-    flex: 1,
+    flex: 1
   },
   header: {
-    backgroundColor: colors.holytosca,
+    backgroundColor: COLORS.anon_primary,
     flexDirection: 'row',
     alignItems: 'center',
     paddingLeft: 20,
     paddingTop: 6,
-    paddingBottom: 7,
+    paddingBottom: 7
   },
   user: {
-    marginLeft: 18,
+    marginLeft: 18
   },
   name: {
     fontFamily: fonts.inter[600],
     fontSize: 14,
-    color: '#fff',
-    lineHeight: 16.94,
+    color: COLORS.white,
+    lineHeight: 16.94
   },
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
-    color: '#fff',
-    lineHeight: 18,
-  },
+    color: COLORS.white,
+    lineHeight: 18
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles have been updated to use the new `COLORS` object for color values. The background color of the header and the text color of the name and time have been updated. This aligns with the PR's goal of standardizing color usage across the application. However, it's important to verify that these new color values are defined in the `COLORS` object and are consistent with the desired look and feel of the application.
```diff
- backgroundColor: colors.holytosca,
+ backgroundColor: COLORS.anon_primary,

- color: '#fff',
+ color: COLORS.white,

- color: '#fff',
+ color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->